src/DataSets/Forms/Sections.component.js

```diff
@@ -0,0 +1,382 @@
+import React from 'react';
+import Translate from 'd2-ui/lib/i18n/Translate.mixin';
+import LinearProgress from 'material-ui/LinearProgress/LinearProgress';
+import FormHelpers from '../../forms/FormHelpers';
+import SearchBox from '../../SearchBox/SearchBox.component';
+import MultipleDataTable from '../../MultipleDataTable/MultipleDataTable.component';
+import LoadingStatus from '../../LoadingStatus/LoadingStatus.component';
+import ObserverRegistry from '../../utils/ObserverRegistry.mixin';
+import Action from 'd2-ui/lib/action/Action';
+import Sidebar from 'd2-ui/lib/sidebar/Sidebar.component';
+//import {Tabs, Tab} from 'material-ui-scrollable-tabs/Tabs';
```

Comment:
The scrollable fork does not forward some styles so I could not customize the tabs. Should be temporal, we need custom styles and scrollable.